pages/apim/installation-guide/installation-guide-migration.adoc

```diff
@@ -0,0 +1,13 @@
+= Migration Guide
+:page-sidebar: apim_sidebar
+:page-permalink: apim_installguide_migration.html
+:page-folder: apim/installation-guide
+:page-toc: true
+
+NOTE: All scripts and configuration files mentioned in below procedures can be found from https://github.com/gravitee-io/release/tree/master/upgrades[GitHub].
+
+include::https://raw.githubusercontent.com/gravitee-io/release/master/upgrades/1.4.0/README.adoc[leveloffset=+1]
```

Comment:
why not include the upgrade README ?
this way we only have to maintain the README and not the doc page.